stage1/init/common/units.go

```diff
@@ -320,10 +320,20 @@ func (uw *UnitWriter) SetupAppIO(p *stage1commontypes.Pod, ra *schema.RuntimeApp
 			// streaming mode brings in a `iomux@.service` before-dependency
 			opts = append(opts, unit.NewUnitOption("Unit", "Requires", fmt.Sprintf("iomux@%s.service", ra.Name)))
 			opts = append(opts, unit.NewUnitOption("Unit", "Before", fmt.Sprintf("iomux@%s.service", ra.Name)))
-			logMode, ok := p.Manifest.Annotations.Get("coreos.com/rkt/experiment/logmode")
+			logMode, ok := p.Manifest.UserAnnotations["coreos.com/rkt/experiment/logmode"]
```

Comment:
I think this was correct before as an annotation, not a user-annotation. Same for the one you added below. See https://github.com/appc/spec/blob/master/spec/pods.md.

Reply:
I think the concern with this is how to pass a pod annotation in rktlet if we use the current rkt CLI. Since we're not using a pod manifest, we'd need to add an option to pass annotations to `rkt run` and `rkt app sandbox`

Reply:
Exaclty, using non-user annotations is just impossible.

Reply:
I'm not sure I'm following, I'm just commenting on stage1 logic here.
It shouldn't matter how you get this information from the user in stage0 (CLI flag, env flag, hardcoded value), you can still pass it as an annotation to stage1.
Unless you are trying to tell me that you want to pass this to stage0 CLI as `--user-annotation=`, in which case I'm saying that you shouldn't (because of spec, and because it will appear back in k8s among user-facing annotations).